packages/mask/shared-ui/locales/en-US.json

```diff
@@ -66,7 +66,7 @@
     "additional_post_box__encrypted_post_pre_ito": "$t(promote_ito) {{encrypted}}",
     "additional_post_box__encrypted_post_pre_file_service_twitter_official_account": "$t(promote_file_service)\n {{encrypted}}",
     "additional_post_box__encrypted_post_pre_file_service": "$t(promote_file_service) {{encrypted}}",
-    "additional_post_box__steganography_post_pre": "This image is encrypted with #mask_io.\n{{random}}\n$t(promote)",
```

Comment:
If you remove this `random`, sending similar pictures might be blocked by Facebook.

Reply:
@DimensionDev/qa  close